framework/Models/Code.py

```diff
@@ -712,7 +712,7 @@ def collectOutputFromDict(self,exportDict,output,options=None):
       @ In, options, dict, optional, dictionary of options that can be passed in when the collect of the output is performed by another model (e.g. EnsembleModel)
       @ Out, None
     """
-    prefix = exportDict.pop('prefix')
+    prefix = exportDict.pop('prefix',None)
```

Comment:
what does this edit do?

Reply:
As we're cleaning stuff up, sometimes 'prefix' has already been removed and can't be "popped" out again.  Adding the `,None` basically means "try popping out `prefix`, but if it isn't there, just give me `None` instead of erroring out."